docs/section19/19-3.md

```diff
@@ -0,0 +1,25 @@
+# Section 19 - Rule 19-3     
+**Schema Version:** 0.0.29
+**Mandatory Rule:** False    
+**Rule ID:** 19-3     
+**Rule Description:** Weather conditions used in sizing runs to determine baseline equipment capacities shall be based either on hourly historical weather files containing typical peak conditions or on design days developed using 99.6% heating design temperatures and 1% dry-bulb and 1% wet-bulb cooling design temperatures.    
+**Rule Assertion:** Options are UNDETERMINED     
+**Appendix G Section:** G3.1.2.2.1      
+**90.1 Section Reference:** None  
+
+**Data Lookup:** None  
+
+**Evaluation Context:** Each RulesetProjectDescription
+
+**Applicability Checks:** None  
+
+**Function Calls:**  None
+
+## Rule Logic:
+- **Rule Assertion:** 
+- All cases: Conduct manual check that rule is met:  `outcome = "UNDETERMINED" and raise_message "Check that the weather conditions used in sizing runs to determine baseline equipment capacities is based either on hourly historical weather files containing typical peak conditions or on design days developed using 99.6% heating design temperatures and 1% dry-bulb and 1% wet-bulb cooling design temperatures."      `  
```

Comment:
I wonder if we should have the outcome be FAIL or UNDETERMINED.
1. Weather-->cooling_design_day_type and heating_design_day_type are selected correctly and we raise_message to tell the reviewer the check in the model that the correct values were used for the location. Outcome == UNDETERMINED
2. Weather-->cooling_design_day_type and heating_design_day_type are NOT selected correctly based on the rules in 90.1 and the outcome ==  FAIL with a raise_message explaining the reason.

Reply:
@claperle This was the line of thinking that I was headed on originally as well! Where I got turned around was the rule definition "Weather conditions used in sizing runs to determine baseline equipment capacities shall be based _**either**_ on hourly historical weather files containing typical peak conditions or on design days developed using 99.6%..." so I agreed with the way you wrote it to return an UNDETERMINED outcome for all projects, though I did not find out why the rule differs from language in 90.1 here.

Reply:
@JacksonJ-KC Where do you see it says in 90.1 that it will be based on either of those? I am not seeing where the language in the rule comes from. Actually reading the language in 90.1 2019 I retract my suggestion. I think this RDS should be what you have now since it does not specify the required design day conditions. 
"Weather conditions used in sizing runs to determine baseline equipment capacities shall be based on design days developed using heating design temperatures, cooling design temperature, and cooling design wet-bulb temperature."